pyproject.toml

```diff
@@ -25,11 +25,10 @@ classifiers = [
     "Intended Audience :: Developers",
 ]
 dependencies = [
-    "ghga-service-commons[all]~=0.6.0",
-    "ghga-event-schemas==0.13.4",
-    "hexkit[mongodb,s3,akafka]~=0.10.2",
-    "httpx~=0.23.3",
-    "typer~=0.7.0"
+    "ghga-service-commons[all]>=1.0.1",
+    "ghga-event-schemas==1.0.0",
```

Comment:
Since every change in the event schema has direct influence on our services we might like to pin that. However, being agnostic to fixes might be worth it: ~=1.0.0